test/e2e/onTTFB-test.js

```diff
@@ -261,8 +261,19 @@ describe('onTTFB()', async function () {
 
       const navEntry = ttfb.entries[0];
       assert.strictEqual(
-        ttfb.attribution.waitingDuration,
-        navEntry.domainLookupStart,
+        ttfb.attribution.redirectDuration,
+        Math.max(
```

Comment:
Nit: I don't think `Math.max()` is needed here (or anywhere you're not subtracting `activationStart`).

Reply:
Fixed.